### ISSUE TITLE
Question/bug: Fusion reactor turns off after server restart and needs reignition

### ISSUE THREAD

**Issue by znorman17** [turbodiesel4598/NuclearCraft#711]
Hey there! I love this mod and really enjoy building the infrastructure (and complexity) for getting fusion reactors up and running.

Background: I have a H-H fusion reactor (toroid size 3) up and running in MC Eternal. My friend and I are using a hosted server which restarts in the middle of the night each day. When I sign in the next day after a restart, when fusion was up and running in the 90's for efficiency before the restart, I noticed the reactor was back to square one for fusion (i.e. temperature was at 0 and my ender energy conduits had all been disconnected from the bottom of the machine).

Just to clarify that my setup is correct: yes, I have plenty of hydrogen being generated (more than I need), plenty of power to get the reactor to be self-sustaining for the electromagnets, and 5000 k/t for active cooling with water connected and working as expected. My entire base is also chunk loaded so none of the machines should ever turn off.

As I mentioned, the reactor works great, but I'm a little concerned about having to restart the reactor every time the server restarts.

Is this a known issue on server restart?

**Comment by znorman17** [comment]
Just a note on this, I made a  second reactor for deuterium and am still experiencing the issue. It seems to be primarily with the connections after the server restarts. I first noticed with the ender energy conduits and also have the same problem with cryo-stabilized fluxducts. If I remove the connecting cable and replace it, then power is correctly pulled into the fusion reactor and it automatically restarts.

**Comment by LarryLava** [comment]
Did you ever solve this issue ? I think I'm experiencing the same on an MC Eternal server. Today reactor was humming along at 82% when I logged off for dinner. Logged back on after an hour, and it had dropped to almost 0 efficiency. Full tanks of hydrogen, 100% active cooling, no redstone signal on core, magnets powered from huge induction matrix. Haven't found a solution yet......

**Comment by znorman17** [comment]
I never did. I started to get bored after spending too much time in my server and this issue made me quit since it was my primary power source. I eventually made two chained reactors then one and the other both stopped working after server restarts. I decided it was way to much work to keep destroying and recreating the reactors.  I'm pretty sure it had to do with things not automatically reconnecting on server restart. Even the fluxducts had to be removed and re-added to connect again to the reactors.

My workaround before I quit was to make a ridiculous amount of ender solar panels and grow/automate the resources for solar panels with mystical agriculture.
